use /safeseh in 32-bit MSVC builds; this is not required for 64-bit

diff -ru openssl.orig/util/mk1mf.pl openssl/util/mk1mf.pl
--- a/openssl.orig/util/mk1mf.pl	2016-03-03 20:22:21.043924505 +0100
+++ b/openssl/util/mk1mf.pl	2016-03-03 20:34:45.015901171 +0100
@@ -488,7 +493,7 @@
 SRC_D=$src_dir
 
 LINK_CMD=$link
-LFLAGS=$lflags
+LFLAGS=$lflags /SAFESEH
 RSC=$rsc \$(SOLARINC)
 
 # The output directory for everything interesting
@@ -511,7 +516,7 @@
 MKDIR=$mkdir
 MKLIB=$bin_dir$mklib
 MLFLAGS=$mlflags
-ASM=$bin_dir$asm
+ASM=$bin_dir$asm /safeseh
 
 # FIPS validated module and support file locations
 
